0-RegEx.md

```diff
@@ -108,7 +178,9 @@ answer:
 
```

Comment:
There's no numbered exercises in this assignment as you received it! Sorry about that!